### PR TITLE
Create note listing page UI

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by abrahamparayil**
Fixes: #1

- 84a5d2e Updated libraries
- 504cf6c Created note listing page UI
- f79c8ed Removed Hero component

### CODE REVIEW

File: app/javascript/src/components/Common/Sidebar/constants.js [hunk 1]
```diff
@@ -1,7 +1,4 @@
-import React from "react";
-
-const NotesIcon = () => <i className="text-2xl ri-file-text-line" />;
-const SettingsIcon = () => <i className="text-2xl ri-settings-2-line" />;
+import { Text, UserCircle, Report } from "@bigbinary/neeto-icons";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Common/Sidebar/constants.js:1]: Please use aliases.

File: app/javascript/src/components/Dashboard/Notes/Filter/index.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import { Plus, Search, Settings } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui/v2";
+import { MenuBar } from "@bigbinary/neetoui/v2/layouts";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/Filter/index.jsx:5]: Please use aliases.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 3]
```diff
@@ -1,53 +1,66 @@
 import React, { useState } from "react";
 
-import { Table } from "neetoui/v2";
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:3]: Please use aliases.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 4]
```diff
@@ -1,53 +1,66 @@
...
-    width: "70%",
-  },
-];
-
-export default function NoteTable({
-  setSelectedNoteIds,
-  notes = [],
-  fetchNotes,
-}) {
+export default function NoteTable({ setShowDeleteAlert }) {
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:8]: Let's not prefix or suffix component or file names or folder names with `Pane, Modal, Form, Table, Page etc`. 
Here, `NotesList` or `ListNotes` is a better name.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 5]
```diff
@@ -1,53 +1,66 @@
...
-            setSelectedNoteIds(selectedRowKeys);
-          }}
-          onRowClick={(_, note) => {
-            setSelectedNote(note);
-            setShowEditNote(true);
-          }}
-          allowRowClick={true}
-        />
+      <div className="m-4 w-full notes-table-height">
+        {[1, 2, 3, 4].map(item => (
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:13]: Please move the note data to a constant inside of a `constants.js` and map over it here.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 6]
```diff
@@ -1,53 +1,66 @@
...
-  },
-];
-
-export default function NoteTable({
-  setSelectedNoteIds,
-  notes = [],
-  fetchNotes,
-}) {
+export default function NoteTable({ setShowDeleteAlert }) {
   const [showEditNote, setShowEditNote] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:9]: const [isEditNotePaneOpen, setIsEditNotePaneOpen] = useState(false);

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 7]
```diff
@@ -1,53 +1,66 @@
...
+                  className="ml-2"
+                  user={{
+                    name: "neeto UI",
+                  }}
+                />
+              </div>
+            </div>
+          </div>
+        ))}
       </div>
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:62]: Please refactor this into a reusable component. 
Create a file `Notes/Card/index.jsx` which acts as the entry point and split the current component into a reusable component.

Comment (karthiknmenon) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:62]: Please create a compound card component as `Card.Header, Card.Body, Card.Footer`.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 8]
```diff
@@ -1,20 +1,23 @@
 import React, { useState, useEffect } from "react";
 
+import { Plus } from "@bigbinary/neeto-icons";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:3]: Please use aliases.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 9]
```diff
@@ -1,20 +1,23 @@
...
 import DeleteAlert from "./DeleteAlert";
+import Filter from "./Filter";
 import NoteTable from "./NoteTable";
 import NewNotePane from "./Pane/CreateNote";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [showMenuBar, setShowMenuBar] = useState(true);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:20]: `const [isMenuBarOpen, setIsMenuBarOpen] = useState(true);`

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 10]
```diff
@@ -1,20 +1,23 @@
...
 
 import DeleteAlert from "./DeleteAlert";
+import Filter from "./Filter";
 import NoteTable from "./NoteTable";
 import NewNotePane from "./Pane/CreateNote";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:19]: Please rename all boolean states properly.

**Review by karthiknmenon — COMMENTED** [review]
@abrahamparayil Please check the suggestions.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx (hunk 4 above)
Comment (abrahamparayil) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx:8]: @karthiknmenon This is the name of the component in `wheel` by default.
Reference: https://github.com/bigbinary/wheel/tree/master/app/javascript/src/components/Dashboard/Notes

**Pull request merged**